source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -0,0 +1,404 @@
+| Preview button   | Preview the currently selected track while pressed               |                                          |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| Star button      | Change the selected track color on short press (next color, or   |                                          |
+|                  | previous if shifted)                                             |                                          |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| Playlist button  | Open or close a context menu for the currently selected track    | On if there is a context-menu open, off  |
+|                  |                                                                  | otherwise                                |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| Mixer FX button  | Toggle third effect (short press) or trigger third effect        | - Dim on if the effect is active         |
+| Mixer FX button  | (long press) or assign the quick effect                          | - Dim on if the effect is active         |
+| Mixer FX button  | of FX select buttons are pressed                                 | - Dim on if the effect is active         |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| FX Select        | Apply effect to all deck on release, if no mixer FX button have  |                                          |
+| button           | been pressed                                                     |                                          |
++------------------+------------------------------------------------------------------+------------------------------------------+
+| Ext              | Apply the current gain as default. This will reset the gain knob.|                                          |
++------------------+------------------------------------------------------------------+------------------------------------------+
+
+
+Looping
+^^^^^^^
+
+===================================================== ==============================================================================
+Control                                               Description
+===================================================== ==============================================================================
+Right Encoder (turn)                                  Double/halve loop size.
+Right Encoder (press)                                 Activate/exit loop of set size from current position
+:hwlabel:`SHIFT` + REV (while no loops are enabled)   Set the loop in point. This will reset the loop out point as well.
```

Comment:
```suggestion
:hwlabel:`SHIFT` + :hwlabel:`REV` (while no loops are enabled)   Set the loop in point. This will reset the loop out point as well.
```